docs/getting-started/edfi-exchange/technology/quick-deploy-of-ed-fi-ods-api-development-server-using-on-premises-server.md

```diff
@@ -0,0 +1,22 @@
+# Quick Deploy of Ed-Fi ODS / API Development Server Using On Premises Server
+
+![MSDF Logo](https://edfidocs.blob.core.windows.net/$web/img/edfi-exchange/technology/msdflogo.png)
+
+## Description
+
+This Powershell script installs the Ed-Fi ODS / API  and Admin App binaries for v2.6, v3,2, v3.3, and v3.4 released on MyGet on a Windows machine.
+
+## Download
+
+* ****Code and Documentation:**** [https://github.com/Ed-Fi-Exchange-OSS/Ed-Fi-BinariesPowerShellInstaller](https://github.com/Ed-Fi-Exchange-OSS/Ed-Fi-BinariesPowerShellInstaller)
```

Comment:
I was worried that the extra asterisk would cause a problem, but the display is fine. We can clean that up later (or never).